spec/robots/assembly/accessioning_initiate_spec.rb

```diff
@@ -3,63 +3,59 @@
 require 'spec_helper'
 
 RSpec.describe Robots::DorRepo::Assembly::AccessioningInitiate do
+  let(:base_url) { 'http://dor-services.example.edu' }
   let(:druid) { 'aa222cc3333' }
-  let(:robot) { Robots::DorRepo::Assembly::AccessioningInitiate.new(druid: druid) }
+
+  subject(:robot) { Robots::DorRepo::Assembly::AccessioningInitiate.new(druid: druid) }
 
   before do
-    allow(Dor::Services::Client).to receive(:initialize_workspace)
-    allow(Dor::Services::Client).to receive(:initialize_workflow)
+    stub_request(:post, "#{base_url}/v1/objects/druid:#{druid}/apo_workflows/accessionWF")
```

Comment:
This test exposes knowledge of the internals of Dor::Services::Client that this code shouldn't know about.  Is there any reason we need this?

Reply:
This is ready for another look, @jcoyne: https://github.com/sul-dlss/assembly/pull/124/files#diff-b104351c23471dd2e1de8be027fa3bf1